bulk-load/src/test/java/no/sikt/nva/data/report/api/etl/loader/BulkLoadHandlerTest.java

```diff
@@ -0,0 +1,75 @@
+package no.sikt.nva.data.report.api.etl.loader;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import no.unit.nva.stubs.FakeContext;
+import nva.commons.logutils.LogUtils;
+import org.junit.jupiter.api.Test;
+
+class BulkLoadHandlerTest {
+
+    @Test
+    void shouldLogSuccessfulLoadingEvent() throws IOException, InterruptedException {
+        final var logger = LogUtils.getTestingAppenderForRootLogger();
+        var responseString = createSuccessResponseString();
+        var httpClient = setUpSuccessfulHttpResponse(responseString);
+        var handler = new BulkLoadHandler(httpClient);
+        handler.handleRequest(null, new ByteArrayOutputStream(), new FakeContext());
+        assertTrue(logger.getMessages().contains(responseString));
+    }
+
+    @Test
+    void shouldLogFailingLoadingEvent() throws IOException, InterruptedException {
+        final var logger = LogUtils.getTestingAppenderForRootLogger();
+        var responseString = createFailingResponseString();
+        var httpClient = setUpFailingHttpResponse(responseString);
+        var handler = new BulkLoadHandler(httpClient);
+        handler.handleRequest(null, new ByteArrayOutputStream(), new FakeContext());
+        assertTrue(logger.getMessages().contains(responseString));
+    }
+
+    private String createFailingResponseString() {
+        return """
```

Comment:
This is made up, I don't know what the response will be — I'm sure we'll find out.